frontend/src/resources/placement-binding.ts

```diff
@@ -13,6 +13,10 @@ export interface PlacementBinding extends IResource {
   apiVersion: PlacementBindingApiVersionType
   kind: PlacementBindingKindType
   metadata: Metadata
+  bindingOverrides?: {
+    remediationAction?: 'inform' | 'enforce' | null
```

Comment:
You can't set an override of "inform". We do accept `Enforce` and `enforce` though.
```suggestion
    remediationAction?: 'Enforce' | 'enforce' | null
```